lib/modules/commentQuickCollapse.js

```diff
@@ -35,3 +107,40 @@ function hideCommentsOnHeaderDoubleClick() {
 		}
 	});
 }
+
+function toggleCommentsOnClickLeftEdge() {
+	const leftEdgeWidth = _.clamp(parseInt(module.options.leftEdgeWidth.value, 10), 0, parseInt((module.options.leftEdgeWidth: any).default, 10)) / 100;
+	const stylesheet = toggleCommentsOnClickLeftEdgeStylesheetTemplate({
+		bColor: module.options.leftEdgeColor.value,
+		hColor: module.options.leftEdgeHoverColor.value,
+		cColor: module.options.leftEdgeCollapsedColor.value,
+		widthI: 2 * leftEdgeWidth,
+		widthO: 2.5 * leftEdgeWidth,
+	});
+
+	addCSS(stylesheet);
+}
+// scrolls on collapse
+function scrollOnCollapse() {
+	$(document.body).on('click', '.expand', (event: MouseEvent) => {
+		// Stops event if condition not met
+		if (click.isProgrammaticEvent(event) || Thing.checkedFrom(event.target).thing.classList.contains('noncollapsed')) {
+			return;
+		}
+		// Scrolls by a default value if top of comment is out of window
+		if ($(event.target).offset().top < $(window).scrollTop()) {
```

Comment:
This and below can probably be replaced with

```
			KeyboardNav.move(
				thing => thing.getClosest(thing.getNextSibling, { direction: 'down' }),
				{ scrollStyle: 'top' }
			);
```

though the `move` function (https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/737b617c4de860ef322a54e6a6267ff66f321d16/lib/modules/keyboardNav.js#L966) needs to be exported first.